ynput/OpenPype
# AYON: Addon settings in OpenPype

## Changelog Description
Moved settings addons to OpenPype server addon. Modified create package to create zip files for server for each settings addon and for openpype addon.

## Additional info
Moved settings from https://github.com/ynput/ayon-addons-settings and photoshop, aftereffects and harmony repository to OpenPype. All the AYON addons contain only settings as it is easier to maintain them when they are in single repository until they're split into separated addons. With this PR will be other repositories archived or modified to point to OpenPype.

The zip files are prepared for new expected structure of uploaded addon zip files on server. Contain `manifest.json` and subdirectory `addon`.

## Testing notes:
1. Run `python ./server_addon/create_ayon_addons.py`
2. Check `./server_addon/packcages/` where should be bunch of zip files
3. Try to upload the files to a server
    - may not work will all server versions, the change is not implemented at the moment of PR creation
